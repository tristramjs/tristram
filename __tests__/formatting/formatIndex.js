// @flow
import createIndexSitemap from '../../src/formatting/formatIndex';

describe('formatIndex', () => {
	const number = 10;
	const hostname = 'computer.me';
	const path = '/here/sitemap';
	test('formatIndexSitemap', () => {
		expect(createIndexSitemap(number, hostname, path)).toMatchSnapshot();
	});
});
