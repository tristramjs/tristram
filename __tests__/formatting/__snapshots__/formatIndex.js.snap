// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`formatIndex formatIndexSitemap 1`] = `
Object {
  "sitemapindex": Object {
    "@xmlns": "http://www.sitemaps.org/schemas/sitemap/0.9",
    "sitemap": Array [
      Object {
        "loc": "https://computer.me/here/sitemap/sitemap-9.xml",
      },
      Object {
        "loc": "https://computer.me/here/sitemap/sitemap-8.xml",
      },
      Object {
        "loc": "https://computer.me/here/sitemap/sitemap-7.xml",
      },
      Object {
        "loc": "https://computer.me/here/sitemap/sitemap-6.xml",
      },
      Object {
        "loc": "https://computer.me/here/sitemap/sitemap-5.xml",
      },
      Object {
        "loc": "https://computer.me/here/sitemap/sitemap-4.xml",
      },
      Object {
        "loc": "https://computer.me/here/sitemap/sitemap-3.xml",
      },
      Object {
        "loc": "https://computer.me/here/sitemap/sitemap-2.xml",
      },
      Object {
        "loc": "https://computer.me/here/sitemap/sitemap-1.xml",
      },
      Object {
        "loc": "https://computer.me/here/sitemap/sitemap-0.xml",
      },
    ],
  },
}
`;
